build/media-suite_compile.sh

```diff
@@ -454,11 +454,13 @@ if [[ $ffmpeg != "no" || $standalone = y ]] && enabled libtesseract; then
 
     _check=(libtesseract.{,l}a tesseract.pc)
     if do_vcs "https://github.com/tesseract-ocr/tesseract.git"; then
-        do_pacman_install docbook-xsl
+        do_pacman_install docbook-xsl nettle
         do_autogen
         _check+=(bin-global/tesseract.exe)
         do_uninstall include/tesseract "${_check[@]}"
-        sed -i "s|Libs.private.*|& -lstdc++|" tesseract.pc.in
+        sed -i -e 's|Libs.private.*|& -lstdc++|' \
+               -e 's|Libs.private.*|& -larchive -lnettle -lregex -lexpat -llzma|' \
+               -e 's|Libs.private.*|& -lzstd -llz4 -lbz2 -lz -lbcrypt -liconv|' tesseract.pc.in
```

Comment:
Instead of the two lines above, do `-e 's|Requires.private.*|& libarchive iconv|'`

Reply:
If that would work then so should have the --libs switch, which I tried. But that complains about lots of missing *.pc files?

Reply:
OK, will check again.

Reply:
It's searching only destdir and not mingw dirs, got it, probably that's why ...

Reply:
On 2nd thought:  PKG_CONFIG_PATH="/local64/lib/pkgconfig:/mingw64/lib/pkgconfig" so actually it shouldn't have complained at all?

Anyway, script is running, but there are some updates ... so currently twiddling my thumbs waiting!

Reply:
OK, it's working, thanks for the help Ricardo! :+1: